book/image-controller.md

```diff
@@ -1,2 +1,125 @@
 # Image Controller
 
+## Overview
```

Comment:
Can you go into the more generic background a little more? Why do we need help setting up container image repositories? One example might be to increase isolation between images produced across various workspaces.